drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -29,6 +29,7 @@ class Forms::FormDefinition < Types::BaseObject
     field :definition, Forms::FormDefinitionJson, null: false
     field :raw_definition, JsonObject, null: false
     field :system, Boolean, null: false
+    field :status, HmisSchema::Enums::FormStatus, null: false, default_value: Hmis::Form::Definition::DRAFT
```

Comment:
Status is non-nullable in the database, so this can be `null: false` (and no default value).

We could implement some behavior whereby it resolves "draft" or "invalid" if the stored status is not recognized. I don't think that's necessary, though, since we know that everything is "draft" right now and we will be validating that going forward.